core/dataflow/src/main/resources/arrow_for_nltk_sentiment.py

```diff
@@ -1,12 +1,12 @@
 import sys
 import pickle
-import csv
+import pyarrow as pa
+import pandas
 
 pickleFullPathFileName = sys.argv[1]
 dataFullPathFileName = sys.argv[2]
 resultFullPathFileName = sys.argv[3]
-inputDataMap = {}
-recordLabelMap = {}
+global inputPb, outputPb
```

Comment:
change the `pb` in `inputPb` `outputPb` to a complete name, what does `pb` mean?

Reply:
Done.